lightning/src/util/config.rs

```diff
@@ -269,6 +269,13 @@ pub struct ChannelConfig {
 	/// [`Normal`]: crate::chain::chaininterface::ConfirmationTarget::Normal
 	/// [`Background`]: crate::chain::chaininterface::ConfirmationTarget::Background
 	pub force_close_avoidance_max_fee_satoshis: u64,
+	/// Sets how many the percent of the channel value the holder_max_htlc_value_in_flight_msat
+	/// will be assigned as.
+	///
+	/// Should be set to a value between 1-100, where the value signals the percantage to be used.
+	///
+	/// Default value: 10.
+	pub holder_max_htlc_value_in_flight_msat_channel_value_percent: u8,
```

Comment:
Maybe `max_inbound_htlc_value_in_flight_percent_of_channel` - I don't like the `holder` terminology so much in public things like this if we can avoid it "inbound_htlc" seems more clear because, well, its an inbound HTLC. That said, I don't really care if the name is long, as long as its descriptive.

Reply:
Also this should be in `ChannelHandshakeConfig`.

Reply:
Ok! Given that we don't serialize the `ChannelHandshakeConfig` in the `Channel` struct, should I implement the channel serialization as follows:
- If the `max_inbound_htlc_value_in_flight_percent_of_channel` has been modified to any other value than the default, we write the `holder_max_htlc_value_in_flight_msat` into the tlv fields (as it's been configured), despite that it will break backwards compatibility?